pysrc/models/metanome_run.py

```diff
@@ -351,22 +383,39 @@ def run_as_compared_csv_line(run: MetanomeRun, baseline: MetanomeRunResults) ->
             fname = fname + '_' + split_metadata[2]
             sampling_rate = sampling_rate[0] + '.' + sampling_rate[1:]
         else:
-            fname, sampling_rate, sampling_method  = sampled_file, '1.0', 'None'
+            fname, sampling_rate, sampling_method = sampled_file, '1.0', 'None'
 
         file_names.append(fname)
         methods.append(sampling_method)
         rates.append(sampling_rate)
 
+    # TODO: Clean up this code block above....
+    # Quick fix to make plots prettier
```

Comment:
Do you want to keep this and the things below like this, or do you want to change it back before merging?

Reply:
That really depends on what we want to do in the future with this.

Looking forward are we going to be mixing sampling methods or just having a single one in each Run?

I'd say lets talk about this on tuesday.